Updated English C courses

apu52 opened the pull request:
## What does this PR do?
Add resource(s) 

## For resources
### Description
I have improved English sources repo. I have added some English C programming courses.
### Why is this valuable (or not)?
This are very valuable courses
### How do we know it's really free?
It's totally free
### For book lists, is it a book? For course lists, is it a course? etc.
It's a course list
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

Ajay-singh1 commented:
LGTM!

Ajay-singh1 reviewed (approved):
LGTM!

apu52 commented:
@eshellman Please merge this PR

eshellman commented:
checking for build problems

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -152,6 +152,7 @@
 * [C Language Tutorial for Beginners (with Notes & Practice Questions)](https://www.youtube.com/watch?v=irqbmMNs2Bo) - Apna College
 * [C Language Tutorial For Beginners In Hindi (With Notes)](https://www.youtube.com/watch?v=ZSPZob_1TOk) - CodeWithHarry
 * [C Language Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtksgReOH3s7R1_cb1QA8vrb) - codeitup
+* [C Language Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtksgReOH3s7R1_cb1QA8vrb) - codeitup
```
duplicate

apu52 commented:
@eshellman I have removed the duplicate link & also have added C programming Course in English .Please merge it

Ajay-singh1 reviewed (commented):
Wait for the maintainer's response

apu52 commented:
> There is nothing new here you just added a duplicate course and removed it.I think this is an invalid PR.

I have added a F programming course in the English course section....please check it

apu52 commented:
![IMG_20231021_121646.jpg](https://github.com/EbookFoundation/free-programming-books/assets/114172928/3e6694e3-8f8d-4706-82d7-db9eea731521)

This course....I have added & also deleted the duplicate course which was not added by me

apu52 commented:
@Ajay-singh1 @eshellman

Ajay-singh1 commented:
> @Ajay-singh1 @eshellman

Wait for Eric.

eshellman commented:
Thanks, and welcome to free-programming-books!
